build.gradle

```diff
@@ -282,11 +283,14 @@ allprojects {
   }
 
   // support for reproducible builds
-  tasks.withType(AbstractArchiveTask).configureEach {
+  tasks.withType(AbstractArchiveTask).configureEach { task ->
     // ignore file timestamps
```

Comment:
Maybe change this comment to "Replace file timestamp with latest git revision date if available"?